app/jobs/jobs/discourse_rss_polling/poll_feed.rb

```diff
@@ -8,8 +8,10 @@ class PollFeed < ::Jobs::Base
       def execute(args)
         return unless SiteSetting.rss_polling_enabled
 
+        @author = User.find_by_username(args[:author_username])
+        return if !@author
```

Comment:
Could be a one-liner (it's the only use of `unless` I find useful)

```suggestion
        return unless @author = User.find_by_username(args[:author_username])
```